apis/v1alpha2/grpcroute_types.go

```diff
@@ -147,7 +147,6 @@ type GRPCRouteSpec struct {
 	//
 	// +optional
 	// +kubebuilder:validation:MaxItems=16
-	// +kubebuilder:default={{matches: {{method: {type: "Exact"}}}}}
```

Comment:
I'm guessing this was accidental removal? Should revert this to avoid API change here.

Reply:
This is being discussed in https://github.com/kubernetes-sigs/gateway-api/pull/2311#discussion_r1295584514

Reply:
Looks like this is actually a reasonable change. The default _behavior_ is to "match all traffic." In a previous state of the GEP, this default did exactly that. However, an empty matcher is now disallowed. However, per the spec, an _unspecified_ GRPCRoute should match all traffic. While this might be a literal API change, it is not a behavioral change. I think this is the right thing to do.